src/modules/dashboard/components/ExpenditurePage/Stages/ClaimFunds/ClaimFunds.tsx

```diff
@@ -139,11 +98,15 @@ const ClaimFunds = ({
             <FormattedMessage {...MSG.claimableNow} />
           </span>
           <div className={styles.valueContainer}>
-            {claimableNow?.map(({ amount, token }) => (
-              <div className={styles.value} key={`${token?.id}-now`}>
-                <Numeral value={amount || 0} /> {token?.symbol}
-              </div>
-            ))}
+            {claimableNow?.map(({ amount, token }, index) => {
+              const key = `${token && 'id' in token ? token?.id : index}-now`;
```

Comment:
I think this could be simplified to:
```suggestion
              const key = `${token?.id ? token.id : index}-now`;
```

Reply:
Also I recently saw a comment (IIRC it was @willm30's) suggesting to use `nanoid` instead of index